streams/src/main/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdater.java

```diff
@@ -295,32 +329,53 @@ private void addTaskToRestoredTasks(final StreamTask task) {
     private final Time time;
     private final ChangelogReader changelogReader;
     private final Consumer<Set<TopicPartition>> offsetResetter;
+    private final boolean manualStart;
```

Comment:
Do we need this boolean flag, or could we just not call `start()` in unit tests?